.semaphore/release/release.yml

```diff
@@ -24,26 +24,9 @@ blocks:
           # Load the github access secrets.  First fix the permissions.
           - chmod 0600 /home/semaphore/.keys/git_ssh_rsa
           - ssh-add /home/semaphore/.keys/git_ssh_rsa
-          # For some reason, /mnt is 100 GB and has a qemu-nbd image file.
-          # Let's delete it and use it for our own purposes (building calico
-          # without running out of space)
```

Comment:
This is probably true for older e1 gen semaphore VMs but not for f1 we currently use. On f1, the 100GB cached container image disk `/dev/sdc1` is mounted directly into `/var/lib/docker`. There is no qemu-nbd image under `/mnt`. I believe the following commands essentially created a folder on the local physical volume and bind mount to the home `calico` folder.